hcn/hcnnetwork_test.go

```diff
@@ -66,3 +66,61 @@ func TestListNetwork(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestAddPolicyNetwork(t *testing.T) {
```

Comment:
rename this to TestAddRemoveRemoteSubnetRoutePolicy

Reply:
Done.